vignettes/delta-method.Rmd

```diff
@@ -104,37 +104,44 @@ P_e(X_1 \beta) &= \frac{d(\text{link}^{-1}(X_1 \beta))}{d(X \beta)} +
 
 # How does the Delta Method Work?
 
-The equations above describe how to calculate predicted levels or effects, but 
-we want to calculate errors on that estimate. To do that, broadly speaking, 
-we'll use a bunch of matrix algebra to run the variance/covariance matrix from 
-the original regression through the gradient of the inverse link function. 
+The equations above describe how to apprxoimate predicted levels or effects, but
+why not just calculate our estimate $P(X\beta)$ directly?
 
```

Comment:
Updated this section adding a few lines showing how J' V J comes from the Taylor Series expansion above, and rearranged some of the explanation below to match.